parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Update Labels for Kubernetes nodes
+
+set -euo pipefail
+
+for kubelet_label in $(echo $KUBELET_NODE_LABELS | sed "s/,/ /g")
+do
+  kubectl label --kubeconfig /var/lib/kubelet/kubeconfig --overwrite nodes $HOSTNAME $kubelet_label
```

Comment:
a bit of logging?

Reply:
Simply ```echo```? I only spotted echos that I would expect are logging by doing a quick glance through the code.

I'll add a few echos here :)

Reply:
Added, two echos. What do you think?

Reply:
Added echos for the new refactored labels.sh

Reply:
Resolving as I assume the echos I added are good enough. I did some testing and found the echos I made in the logs for the service on the node which were quite helpful :)